File: src/netcore-api/SIL.XForge.WebApi.Server/Models/InputSystem.cs
namespace SIL.XForge.WebApi.Server.Models
{
    public class InputSystem
    {
        public InputSystem(string tag = "qaa", string name = "", string abbr = "")
        {
            Tag = tag;
            Abbreviation = abbr;
            LanguageName = name;
        }

        public string Abbreviation { get; set; }
        public string Tag { get; set; }
        public string LanguageName { get; set; }
        public bool IsRightToLeft { get; set; }
    }
}

File: src/Api/Model/Shared/InputSystem.php
<?php

namespace Api\Model\Shared;

class InputSystem
{
    public function __construct($tag = "qaa", $name = "", $abbr = "")
    {
        $this->tag = $tag;
        $this->abbreviation = $abbr;
        $this->languageName = $name;
        $this->isRightToLeft = false;
        $this->cssFontFamily = "";
    }

    /** @var string */
    public $abbreviation;

    /** @var string */
    public $tag; // RFC5646 tag e.g. qaa-x-lang

    /** @var string */
    public $languageName;

    /** @var boolean */
    public $isRightToLeft;

    /** @var string */
    public $cssFontFamily;
}
